### DISCUSSION TITLE
Excluding my own mobile (iOS) visits

### DISCUSSION THREAD

**Discussion by sil-so** [plausible/analytics#3250] [Anything Else]
Hey all, is there a way to exclude my own mobile (iOS - Safari mostly) visits from the stats? I couldn't find anything related to this in the docs and the latest discussions are from a few years ago so I'm wondering if there are developments passed on this matter. I need to check my site regularly and don't want to mess up my statistics. Thanks

**Comment by metmarkosaric** [comment]
hi @silvan-soeters! we normally recommend using an adblocker to exclude your own visits. see https://plausible.io/docs/excluding

**Comment by metmarkosaric** [comment]
**Update**: It’s now easy to exclude your internal traffic from being counted in your stats. See how here: https://plausible.io/docs/excluding
